### PR TITLE
Pandacss anchor buttons

### PR DESCRIPTION
**Pull request by yangchristina**
#2170 
This is the last of the element selectors in App.css
- remove unused `ul ul.context-chain` from App.css
- removed className='tutorial-text', which doesn't have any styles

### CODE REVIEW

**Comment by yangchristina** [comment]
I failed a puppeteer test earlier: https://github.com/cybersemics/em/actions/runs/10983149534/job/30492188860?pr=2407

I make some small changes that wouldn't make any difference really, and the tests are passing now. I also couldn't find any connection between the failing snapshot and the code in my last commit, which seems odd.

File: src/components/AppComponent.tsx [hunk 1]
```diff
@@ -260,7 +189,6 @@ const AppComponent: FC = () => {
 
   return (
     <div className={componentClassNames}>
-      <GlobalStyles styles={globalStyles} />
```
Comment (trevinhofmann) [inline src/components/AppComponent.tsx:263]: Awesome! 🎉

File: src/App.css [hunk 2]
```diff
@@ -23,10 +23,6 @@
   z-index: var(--z-index-scroll-zone);
 }
 
-ul ul.context-chain {
-  margin-left: 1.6em;
-}
```
Comment (trevinhofmann) [inline src/App.css:28]: While we're removing the unused `.context-chain`, now would be a good time to clean up the other references to it by removing these comments? https://github.com/cybersemics/em/blob/25cbb95a8ac01a42540d93a459eaef5e9d4f1498/src/App.css#L685-L713

**Review by trevinhofmann — CHANGES_REQUESTED** [review]
Aside from the one suggestion about `.context-chain` cleanup, this pull request looks good to me. Thanks, @yangchristina!

**Review by trevinhofmann — APPROVED** [review]

**Review by raineorshine — APPROVED** [review]

**Pull request merged**